packages/features/calendars/DatePicker.tsx

```diff
@@ -135,6 +136,9 @@ const Days = ({
     const date = browsingDate.set("date", day);
     days.push(date);
   }
+
+  const isMobile = useMediaQuery("(max-width: 768px)");
+
```

Comment:
note: i was initially looking for something that would use our tailwind config but sometimes you want to trigger this on a pixel-perfect level and not based on tailwind `sm` or `md` .. breakpoints so we'll keep this pixel based